### DISCUSSION TITLE
Support for double-page spreads?

### DISCUSSION THREAD

**Discussion by tengento** [sumatrapdfreader/sumatrapdf#2341] [Q&A]
Hi, I am using Sumatra to view comic book rars. These comics often will have a double-page spread, but in the rar the two pages are combined as a single jpeg. They look fine in single-page mode, but in facing mode (the main mode I use for majority of each book) Sumatra ends up showing 3 pages. Is there any way to have Sumatra show these wider pages in single-page view, then return to facing view when the next two pages are the same, narrower width?

Here is what it looks like in facing view (manga mode) with a double-page, then two normal pages, then another double page. 
![1](https://user-images.githubusercontent.com/89487229/153703084-64b6c156-7566-4cac-8b61-1fd193fba3e9.png)
![2](https://user-images.githubusercontent.com/89487229/153703086-a5cbef0d-3f3b-44f5-83e9-fd83dce09d7b.png)

**Comment by GitHubRulesOK** [comment]
It is a historic issue since the double spreads are simply one page so 2 looks like 3 or even 4. 
One solution (common to many comics) is to split the doubles in half to ensure all pages are paired but sometimes causes other issues such as when swapping right / left modes the doubles may seem visually out of sync (bad authorship). 
SumatraPDF  setting allow  for such comics as you can close the gap to make the half doubles appear stiched or change from facing to book to allow for natural pairing. That still does not compensate for bad splits where the synced pages are not correctly ordered by the author. 
Some other dedicated comic readers are able to detect the comic book speads and change behavior, but SumatraPDF is not only for comics so if Artworks or Engineering Drawings (just like double spreads) were split into two sequential pages there would be much screaming about that splitting, thus best to very simply count each image as = 1 page.

**Reply by tengento** [reply]
Would it be a good idea to have double spread support as a toggle, similar to the manga mode toggle?

**Comment by Kuroonehalf** [comment]
The clear solution to me seems to be to just look at the aspect ratio of the images being loaded. If the first is landscape (width > height), then it's a spread. If the first image is portrait but the second is landscape, show image 1 alone, and then on next advance you show the spread.

I would love to use SumatraPDF for my manga reading, but until this feature is in it's just too cumbersome to deal with this issue.
